### PR TITLE
OPENEUROPA-1906: Initial module scaffolding.

### PR DESCRIPTION
**Pull request by imanoleguskiza**

### CODE REVIEW

**Comment by voidtek** [comment]
The drone file should be updated (https://github.com/openeuropa/drupal-module-template/blob/master/.drone.yml)

File: runner.yml.dist [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    profile: "minimal"
+  database:
+    host: "mysql"
+    port: "3306"
+    name: "oe_oembed"
+    user: "root"
+    password: ""
+  post_install:
+    # Prepare the instance.
+    - "./vendor/bin/drush en toolbar -y"
```
Comment (dxvargas) [inline runner.yml.dist]: Should we use the `admin_toolbar` instead? I can update the `drupal-module-template` if so.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# OpenEuropa oe_oembed
+
+The OpenEuropa oe_oembed provides ...`[One Paragraph of project description goes here]`
```
Comment (dxvargas) [inline README.md]: Should be updated with meaningful information.

File: runner.yml.dist (hunk 1 above)
Comment (upchuk) [reply runner.yml.dist]: No. That is in EWCMS.

File: README.md (hunk 2 above)
Comment (upchuk) [reply README.md]: Yes, thanks!

**Pull request merged**